javascript-forms/form-events.js

```diff
@@ -0,0 +1,27 @@
+document.querySelectorAll('form');
+
+// focus event
+
+document.addEventListener('focusin', handleFocus);
+
+function handleFocus(event) {
+  console.log('focus event fired');
+  console.log('event.target.name', event.target.name);
+}
+
+// blur event
+
+document.addEventListener('focusout', handleBlur);
+
+function handleBlur(event) {
+  console.log('blur event fired');
+  console.log('event.target.name', event.target.name);
+}
+
+// input event
+
+document.addEventListener('input', handleInput);
+
+function handleInput(event) {
+  console.log('Value of', event.target.name, ':', event.target.value);
+}
```

Comment:
You shouldn't be querying for the `form`, or adding events to the `document`. You should be querying for each of our form control elements individually, and attaching the events to them